### PR TITLE
Updated to new standards

### PR DESCRIPTION
**Pull request by mueschm**
## Overview
Updates to the new flow.

## Changes
1. Remove the big text block
2. Add a db and backend
3. Form shows right away now
4. Fixed some styling

## Pictures
chrome_m4Tucj9qEW

chrome_cTqc6g3SsB

## TODO List
Before merging, a PR will need
- [ ] Signoff from product if there are UX changes 
- [x] Signoff from one other engineer at Architect.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,108 +12,111 @@
...
+### Make your own changes
 
-# Adding a dependency
-Using a microservice framework allows you to decouple an application and alleviate many of the problems that come with a monolithic architecture. Architect provides first class support for microservces through the use of dependencies.
+This application's `architect.yml` file contains a `debug` block that enables hot-reloading for each service
+within the component. That means you can make changes to the source code, which will automatically apply to the environment. This lets you quickly iterate and see your changes without restarting the
+application stack.
 
-This example application is configured to connect to the [Node.js Starter Project](https://github.com/architect-templates/node-rest-api) REST API as a [dependency](https://docs.architect.io/components/dependencies/), but you can modify the `architect.yml` file to connect to any REST API backend as an Architect Component.
+Give it a try! Search inside your project for "Favorite Movie” and change this string to “Favorite Pizza.” Once you saved
```
Comment (ryan-cahill) [inline README.md:80]: This string doesn't exist on the page

image

**Review by ryan-cahill — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mueschm) [reply README.md:80]: Fixed
image

**Review by ryan-cahill — CHANGES_REQUESTED** [review]
Running locally looks good, but I ran into this issue on a remote deployment

**Review by ryan-cahill — APPROVED** [review]
Looks like the issue that I mentioned is only on remote deployments from local because of the cert. I'll make a ticket but this looks good to go

`architect dev`
image

`architect dev --debug=false`
image

`architect deploy nuxt -e example-environment --auto-approve`
image

**Review by Zach-at-Arch — CHANGES_REQUESTED** [review]
Error handling is janky - it's just throws a 409, and I need to refresh the page to see the application again. It should let the user know what needs to change (invalid rating input, missing fields, etc.) instead of throwing an error page.

Beyond that, everything else is good - hot reloading works smoothly and the application started up without a hitch.

**Comment by mueschm** [comment]
Added in error message.
image

**Review by Zach-at-Arch — APPROVED** [review]

**Pull request merged**